lib/zlib.js

```diff
@@ -171,11 +176,30 @@ class Zlib extends Transform {
     this._opts = opts;
     this._chunkSize = opts.chunkSize || constants.Z_DEFAULT_CHUNK;
 
-    if (opts.flush && isInvalidFlushFlag(opts.flush)) {
-      throw new RangeError('Invalid flush flag: ' + opts.flush);
+    if (opts.flush) {
+
+      if (!isNumber(opts.flush)) {
+        throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'flush',
+          'number');
```

Comment:
Alignment here too. I will stop pointing this out. Running `make lint` should catch these.